src/config/app.config.ts

```diff
@@ -0,0 +1,3 @@
+export default () => ({
+  appSecret: process.env.JWT_SECRET ?? "Cophr_appSecret",
```

Comment:
is it secure to provide a default secret? may be a potential security issue

Reply:
I originally use for jest test because when the backend init (src/config/env.validation.ts will validate JWT_SECRET is not empty.).

But You are readly. I change the jest test and generate swagger use mock environment variable.

[refactor: set mock environment variable](https://github.com/Cophr/backend/pull/10/commits/da452df74e15060adab9733d4ff86005cf39dcc2)
[refactor: jest test use mock environment variable](https://github.com/Cophr/backend/pull/10/commits/23ca097eeb015fbf28aab238a46a983279aa7105)